pkg/util/paramtable/component_param.go

```diff
@@ -1202,6 +1203,15 @@ Fractions >= 1 will always sample. Fractions < 0 are treated as zero.`,
 	}
 	t.OtlpSecure.Init(base.mgr)
 
+	t.OtlpHeaders = ParamItem{
+		Key:          "trace.otlp.headers",
+		Version:      "2.4.0",
```

Comment:
NIT: this version field stands for which milvus version start to support this param item. In this case we shall make it `2.6.0` or `2.5.15` if you wanna make it work on 2.5 branch